src/test/ui/methods/issues/issue-90315.rs

```diff
@@ -1,7 +1,54 @@
 fn main() {
-  let arr = &[0,1,2,3];
-  for _i in 0..arr.len().rev() { //~ERROR not an iterator
-     // The above error used to say “the method `rev` exists for type `usize`”.
-     // This regression test ensures it doesn't say that any more.
-  }
+    let arr = &[0, 1, 2, 3];
+    for _i in 0..arr.len().rev() { //~ERROR not an iterator
+        // The above error used to say “the method `rev` exists for type `usize`”.
+        // This regression test ensures it doesn't say that any more.
+    }
+
+    // Test
+    for i in 1..11.rev() {
+        //~^ ERROR not an iterator
+        // Suggest add the missing parentheses for range
```

Comment:
```suggestion
        //~| HELP surround the range in parentheses
```
Maybe this is better for all occurrences of this statement?